### ISSUE TITLE
SCRAM-SHA-1-PLUS support

### ISSUE THREAD

**Issue by COM8** [UWPX/UWPX-Client#35]
## Bevor you create a new issue:

```
- [x] I searched for similar issues and did not find one
- [x] I'm using the latest version available in the [Windows Store](https://www.microsoft.com/store/apps/9NW16X9JB5WV)
```

## I'm submitting a...:

  
- Feature request (UserVoice request: )

## Current behavior:

No SCRAM-SHA-1-PLUS support.

## Expected behavior:

SCRAM-SHA-1-PLUS support.

## Environment:

```
App Version(s): 
v.X.Y.Z <!-- Which version of the App are you using (e.g. v.0.2.0) -->

Windows 10 Version Number: <!-- https://en.wikipedia.org/wiki/Windows_10_version_history -->
- [ ] 1809
- [ ] 1803
- [ ] 1709
- [ ] 1703
- [ ] 1607
- [ ] 1511
- [ ] 1507
- [ ] Insider Build (build number: )
- [ ] Misc:

Device form factor:
- [ ] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub

Where did you got the APP from?
- [ ] [Windows Store](https://www.microsoft.com/store/apps/9NW16X9JB5WV)
- [ ] Self-build, using a provided release source
- [ ] Self-build, repo cloned at [dd.mm.yyy] <!-- When did you clone the repo! -->
- [ ] Misc, got it from... <!-- Please tell us your source! -->

```

**Comment by Neustradamus** [comment]
- https://tools.ietf.org/html/rfc8600 since 2019-06-21: https://mailarchive.ietf.org/arch/msg/ietf-announce/suJMmeMhuAOmGn_PJYgX5Vm8lNA

**Comment by COM8** [comment]
Ohm... I don't really get what you want to tell me with this paper.  
I read the first 5 chapters and don't get it.  
It says in the introduction: "`[...] primarily for the purpose of communication among Computer Security Incident Response Teams and associated entities.`"

And it defines an so called `XMPP-Grid` which "`[...] collect and distribute security incident reports and other security-relevant information among network platforms[...]`"

Where is the connection to UWPX? Or is it just a cool paper you wanted to share?

So please talk to me and give me a little bit more information. I can't read minds (at the moment at least) :wink:

**Comment by Neustradamus** [comment]
It is about support of:
- SCRAM-SHA-256(-PLUS)
- and the old SCRAM-SHA-1(-PLUS)

**Comment by COM8** [comment]
And this means?

**Comment by Neustradamus** [comment]
It is for XMPP connections...
Have you progress on it?

**Comment by COM8** [comment]
Closing this since all required information are available in #51.
